merkle/src/store.rs

```diff
@@ -1,33 +1,71 @@
 use failure::Error;
-use merkle::Element;
+use merkle::{Element, next_pow2};
 use positioned_io::{ReadAt, WriteAt};
-use std::fs::File;
+use std::fs::{File, OpenOptions};
+use std::path::{Path, PathBuf};
 use std::marker::PhantomData;
 use std::ops::{self, Index};
 use tempfile::tempfile;
 
 pub type Result<T> = std::result::Result<T, Error>;
 
+
+#[derive(Clone, Debug)]
```

Comment:
we should make this serializable and deserializable using `serde` so we can easily store these if we want to